doc/releases/changelog-0.40.0.md

```diff
@@ -30,19 +30,78 @@
 
 <h4>Enhanced QSVT functionality 🤩</h4>
 
-* New functionality to calculate angles for QSP and QSVT has been added. This includes the function `qml.poly_to_angles`
-  to obtain angles directly and the function `qml.transform_angles` to convert angles from one subroutine to another.
+* New functionality to calculate angles for QSP and QSVT has been added.
   [(#6483)](https://github.com/PennyLaneAI/pennylane/pull/6483)
 
-* The `qml.qsvt` function has been improved to be more user-friendly. Old functionality is moved to `qml.qsvt_legacy`
-  and it will be deprecated in release v0.40.
-  [(#6520)](https://github.com/PennyLaneAI/pennylane/pull/6520/)
+  This includes the function :func:`qml.poly_to_angles <pennylane.poly_to_angles>` to obtain angles directly, given a polynomial:
+  ```pycon
+  >>> poly = [0, 1.0, 0, -1/2, 0, 1/3]
+  >>> qsvt_angles = qml.poly_to_angles(poly, "QSVT")
+  >>> print(qsvt_angles)
+  [-5.49778714  1.57079633  1.57079633  0.5833829   1.61095884  0.74753829]
+  ```
+  And :func:`qml.transform_angles <pennylane.transform_angles>` can be used to convert angles from one subroutine to another:
+  ```pycon
+  >>> qsp_angles = np.array([0.2, 0.3, 0.5])
+  >>> qsvt_angles = qml.transform_angles(qsp_angles, "QSP", "QSVT")
+  >>> print(qsvt_angles)
+  [-6.86858347  1.87079633 -0.28539816]
+  ```
+
+* The :func:`qml.qsvt <pennylane.qsvt>` function has been improved to be more user-friendly.
+  [(#6520)](https://github.com/PennyLaneAI/pennylane/pull/6520)
   [(#6693)](https://github.com/PennyLaneAI/pennylane/pull/6693)
 
-* New `qml.GQSP` template has been added to perform Generalized Quantum Signal Processing (GQSP).
-  The functionality `qml.poly_to_angles` has been also extended to support GQSP.
+  Now, block encoding and angle computation are all handled automatically:
+  ```python
+  # P(x) = -x + 0.5 x^3 + 0.5 x^5
+  poly = np.array([0, -1, 0, 0.5, 0, 0.5])
+  hamiltonian = qml.dot([0.3, 0.7], [qml.Z(1), qml.X(1) @ qml.Z(2)])
+  
+  dev = qml.device("default.qubit")
+  @qml.qnode(dev)
+  def circuit():
+      qml.qsvt(hamiltonian, poly, encoding_wires=[0], block_encoding="prepselprep")
+      return qml.state()
+  
+  matrix = qml.matrix(circuit, wire_order=[0, 1, 2])()
+  ```
+  ```pycon
+  >>> print(matrix[:4, :4].real)
+  [[-0.1625  0.     -0.3793  0.    ]
+   [ 0.     -0.1625  0.      0.3793]
+   [-0.3793  0.      0.1625  0.    ]
+   [ 0.      0.3793  0.      0.1625]]
+  ```
+
+* A new :class:`qml.GQSP <pennylane.GQSP>` template has been added to perform Generalized Quantum Signal Processing (GQSP).
```

Comment:
I recommend adding a bit more background in this entry by including a succinct description of what GQSP is doing. E.g., 

> Similar to QSVT, GQSP is an algorithm that polynomially transforms an input unitary operator. The theoretical details of this algorithm are described in https://arxiv.org/abs/2308.01501. 
> Using `qml.GQSP` in a circuit requires tthree inputs: 
> - `unitary`: the unitary operator to be encoded
> - `angles`: phase angles that define the polynomial transformation
> - `control`: The control qubit that is needed to encode the polynomial transformation
> ...